acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,324 @@
+
+class FingerprintUpdate(FingerprintChange, tag="U"):
+    hashes: bytes
+
+
+class FingerprintDelete(FingerprintChange, tag="D"):
+    pass
+
+
+class NatsFingerprintUpdateReceiver(FingerprintUpdateReceiver):
+
+    def __init__(self, nc: nats.NATS, stream_name: str):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)